requirements/requirements_doc.txt

```diff
@@ -1,5 +1,5 @@
 Sphinx==5.3.0
-ansys-sphinx-theme==0.8.2
+ansys-sphinx-theme==0.9.1
```

Comment:
```suggestion
ansys-sphinx-theme==0.9.2
```
New version available... should it solve the problems?

Reply:
Sadly, no. I had to perform that hot-fix after realizing that there was an issue with the typing. It was very werid.

Reply:
But we need to apply that version for sure.